modules/broker-core/pom.xml

```diff
@@ -51,6 +51,11 @@
             <groupId>io.ballerina.messaging</groupId>
             <artifactId>broker-metrics</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.ballerina.messaging</groupId>
+            <artifactId>broker-event-publisher</artifactId>
+            <version>0.970.6-SNAPSHOT</version>
```

Comment:
No need to specify the version here because this is a module in the project. Even when you need to specify a version its better to do it through a property which can be defined in the main pom.